crates/subspace-runtime/src/lib.rs

```diff
@@ -145,6 +145,19 @@ const ERA_DURATION_IN_BLOCKS: BlockNumber = 2016;
 
 const EQUIVOCATION_REPORT_LONGEVITY: BlockNumber = 256;
 
+/// Initial tx range = U256::MAX / INITIAL_DOMAIN_TX_RANGE.
+const INITIAL_DOMAIN_TX_RANGE: u64 = 10;
+
+/// Tx range is adjusted every DOMAIN_TX_RANGE_ADJUSTMENT_INTERVAL blocks.
+const TX_RANGE_ADJUSTMENT_INTERVAL_BLOCKS: u64 = 100;
+
+/// Expected bundles per slot.
+const EXPECTED_BUNDLES_PER_SLOT: u64 = 1;
```

Comment:
Sorry for late notice, I realized that this should also depend on `bundle_slot_probability` of the domain ([this](https://github.com/subspace/subspace/blob/cf761018293e1a354b72ce47acd17cca61dae458/crates/sp-domains/src/lib.rs#L180), so `EXPECTED_BUNDLES_PER_SLOT = bundle_slot_probability.1`
@liuchengxu Could you confirm?

Reply:
yeap. Should use the domain specific slot probability

Reply:
Yes, you're right. @rahulksnv Due to the DomainRegistry is not yet implemented in pallet-domains, we can make it as a TODO and resolve it later.